content/blog/scheduling-serverless/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Scheduling Serverless"
+date: 2020-02-25
+meta_desc: "Automate maintenance tasks by writing and scheduling serverless functions"
+meta_image: meta.png
+authors:
+    - cyrus-najmabadi
+tags:
+    - serverless
+    - lambda
+---
+
+Scheduling events has long been an essential part of automation; many tasks need to run at specific times or intervals. You could be checking StackOverflow for new questions every 20 minutes or compiling a report that is emailed every other Friday at 4:00 pm. Today, many of these tasks can be efficiently accomplished in the cloud. While each cloud has its flavor of scheduled functions, this post steps you through an example using [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) with the help of Pulumi.
+
+<!--more-->
+
+## Taking out the trash
+
+Let's assume we have an S3 Bucket similar to the Recycle Bin on your desktop. It contains discarded items that we might not be ready to delete yet, but because we are paying for storage, we want to empty this bucket every Friday at 6:00 pm EST.
+
+```typescript
+import * as aws from "@pulumi/aws";
+import { ObjectIdentifier } from "aws-sdk/clients/s3";
+
+// Create an AWS resource (S3 Bucket)
+const trashBucket = new aws.s3.Bucket("trash");
+```
+
+We’re now ready to create our Lambda function and schedule it. Pulumi simplifies creating Lambda functions by allowing us to provide a handler function to an event such as `cloudwatch.onSchedule`. To learn more, check out our [Lambdas as Lambdas](https://www.pulumi.com/blog/lambdas-as-lambdas-the-magic-of-simple-serverless-functions/) post. Let’s create a `cloudwatch.EventRuleEventHandler` async function that will get all the items from the trash bucket and delete them.
+
+```typescript
+// A handler function that will list objects in the bucket and bulk delete them
+const emptyTrash: aws.cloudwatch.EventRuleEventHandler = async (
```

Comment:
Can you apply four-space indentation to this code? That is generally what we use.